tekton/src/pipelines/gitops/gitops-mas-apps.yml.j2

```diff
@@ -209,6 +209,24 @@ spec:
     - name: db2_timezone
       type: string
       default: ""
+    - name: db2_backup_bucket_name
+      type: string
+      default: ""
+    - name: db2_backup_bucket_endpoint
+      type: string
+      default: ""
+    - name: db2_backup_bucket_access_key
+      type: string
+      default: ""
+    - name: db2_backup_bucket_secret_key
+      type: string
+      default: ""
+    - name: db2_backup_notify_slack_url
+      type: string
+      default: ""
+    - name: db2_backup_icd_auth_key
+      type: string
+      default: ""
```

Comment:
We shouldn't have any secrets on the pipelines and tasks directly. These should comes from the secure-properties secret that is used in the envFrom section of the task